src/actions/org.js

```diff
@@ -1,19 +1,46 @@
-import { addOrganization, deleteOrganisation, getOrganisation} from '../utils';
+/* eslint-disable brace-style */
+import { db } from '../utils';
 import { showLoading, hideLoading } from 'react-redux-loading-bar';
+import { Redirect } from 'react-router-dom';
+import { alerts } from './alerts';
 
-export const ADD_ORGANISATION = 'ADD_ORGANISATION';
-export const DELETE_ORGANISATION = 'DELETE_ORGANISATION';
-export const GET_ORGANISATION = 'GET_ORGANISATION';
+export const orgActions = {
+  handleAddOrg,
+  handleDelOrg,
+  handleGetOrg
+};
 
 function addOrg(org) {
-    console.log('addOrg',org)
+  console.log('addOrg',org);
   return {
     type: ADD_ORGANISATION,
     org,
   };
 }
 
-export function handleAddOrg(org) {
+function handleAddOrg(org) {
+  return dispatch => {
+    dispatch(request(org));
+
+    db.addOrganization(org)
+        .then(
+            org => {
+              dispatch(success());
+              <Redirect to='/app/list/members'/>;
+              dispatch(alerts.success('Registration successful'));
+            },
+            error => {
+              dispatch(failure(error));
+              dispatch(alerts.error(error));
+            }
+        );
+  };
+  function request(user) { return { type: userConstants.REGISTER_REQUEST, user } }
+  function success(user) { return { type: userConstants.REGISTER_SUCCESS, user } }
+  function failure(error) { return { type: userConstants.REGISTER_FAILURE, error } }
+}
+
+export function handleAddOr(org) {
```

Comment:
is his intentionally different from L#21

Reply:
that should have been removed, already done